src/Message.module.css

```diff
@@ -0,0 +1,18 @@
+.message {
+	line-height: 1.5;
+	font-weight: 400;
+	background-color: #fff;
```

Comment:
so the question here still is if we can somehow derive something like this message background-color from a theme that we generate from the colorScheme in the webchat codebase. The way I understood you is that we then need to set each css variable that is using this "theme color" manually, is that correct? 

Ideally there would be some sort of templating we could use where one variable is used in another variable if it is set. Then we would just have to set one variable from the webchat code and can reuse. What do you think?

Reply:
That's a good point, have not yet come to that. What we can do is to have some `theme.css` file which is imported in some common ancestor component.
``` css
:root {
    --webchat-primary-color: var(--webchat-primary-color, #00f);
    --webchat-message-bubble: var(--webchat-primary-color);
}
```
Then we can have a single self-documented place for all variable used and 
also have inheritance + the way to escape it. 

I would leave the topic how we map theme in webchat to them out. As it's out of scope for this repo.
There are several ways to do it, even with inegration to the current theme solution, see:

https://mui.com/material-ui/experimental-api/css-theme-variables/migration/

Reply:
I am fine with leaving it out for now but we'll need it for the release for sure!